cosmic-settings/src/pages/desktop/mod.rs

```diff
@@ -31,9 +37,21 @@ impl Default for Page {
             },
         );
```

Comment:
This can be `CosmicTk::config().ok()`

Reply:
Since I removed the `cosmic_tk` attribute, I'm closing this suggestion.